### PR TITLE
Converting the Display Magento error report number KB to devdocs

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

This pull request converts this KB article to a DevDoc article: https://support.magento.com/hc/en-us/articles/360018126211

## Affected DevDocs pages

http://devdocs.magento.com/guides/v2.3/cloud/cdn/cloud-fastly-custom-response.html#display-magento-error-report-number

### CODE REVIEW

File: guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md [hunk 1]
```diff
@@ -143,6 +143,61 @@ To edit the WAF error page:
...
+      nslookup {your_project_id}.ent.magento.cloud
+      ```
+
+   -  Pro plan users, Integration environment; Starter plan users, all environments:
+
+      ```bash
+      nslookup gw.{your_region}.magentosite.cloud
+      ```
+
+1. Add your application domain and IP address to the hosts file on your local machine:
```
Comment (hguthrie) [inline guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]: We don't use colons at the end of a step, even with a command.
Also, not sure why using "local machine".
We already have a problem keeping consistent between "local workstation" and "local workspace". Need to stick with one of them.

File: guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md [hunk 2]
```diff
@@ -143,6 +143,61 @@ To edit the WAF error page:
...
+
+1. Retrieve the IP address of your store:
+
+   -  Pro plan users, Staging and Production environments:
+
+      ```bash
+      nslookup {your_project_id}.ent.magento.cloud
+      ```
+
+   -  Pro plan users, Integration environment; Starter plan users, all environments:
```
Comment (hguthrie) [inline guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]: Consistency.
In many other Cloud guide places, we say "For Pro Staging and Production environments..." and "For Started environments and Pro Integration environment..."

File: guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md [hunk 3]
```diff
@@ -143,6 +143,61 @@ To edit the WAF error page:
 
    -  On the Cache Management page, click **Flush Magento Cache**.
 
+## Display Magento error report number
+
+By default, Fastly hides all Magento errors behind the _503 Service Unavailable_ error. To display the Magento error log report number so that you can find and review the error details in the logs, open the website omitting Fastly using these steps:
+
+1. Retrieve the IP address of your store:
+
+   -  Pro plan users, Staging and Production environments:
```
Comment (meker12) [inline guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]:
```suggestion
   -  For Pro Staging and Production environments:
```

File: guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md (hunk 1 above)
Comment (meker12) [reply guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]: We do use colons at the end of steps, but I noticed that you have been removing them.  I think when the step is followed by a command that is carrying out the step, a colon makes sense.  I wonder what the style guide says.  The style guide doesn't say anything.  Google style guide procedure shows colon as recommended in the context of a specific example. 

I notice that this convention of using the colon is followed in other DevDoc Guides.

File: guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md (hunk 2 above)
Comment (meker12) [inline guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]:
```suggestion
   -  For Pro Integration environments and Starter environments:
```

File: guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md (hunk 1 above)
Comment (meker12) [inline guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]:
```suggestion
1. Add your application domain and IP address to the hosts file on your local workstation:
```

**Comment by meker12** [comment]
running tests

**Pull request merged**